examples/hello-world/hello_world.go

```diff
@@ -0,0 +1,78 @@
+/* This file is part of VoltDB.
+ * Copyright (C) 2008-2016 VoltDB Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with VoltDB.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package main
+
+import (
+	"fmt"
+	"github.com/VoltDB/voltdb-client-go/voltdbclient"
+	"log"
+)
+
+func main() {
+	client := voltdbclient.NewClient("", "")
+	if err := client.CreateConnection("localhost:21212"); err != nil {
+		log.Fatal("failed to connect to server")
+	}
+	defer func() {
+		if client != nil {
+			client.Close()
+		}
+	}()
+
+	// rows to insert
+	rows := make([][]string, 5)
+	rows[0] = []string{"Hello", "World", "English"}
+	rows[1] = []string{"Bonjour", "Monde", "French"}
+	rows[2] = []string{"Hola", "Mundo", "Spanish"}
+	rows[3] = []string{"Hej", "Verden", "Danish"}
+	rows[4] = []string{"Ciao", "Mondo", "Italian"}
+	for _, row := range rows {
+		insertData(client, row[0], row[1], row[2])
```

Comment:
if hello_world is executed second time it crashes. maybe truncate before insert?